src/exports.jl

```diff
@@ -566,6 +568,7 @@ export weights
 export with_unicode
 export zero
 export zero!
+export submul!
```

Comment:
this should be moved to the alphabetical correct place

Reply:
ouch, how did that happen!?!? thank you